server/sandbox_run_linux.go

```diff
@@ -353,12 +353,6 @@ func (s *Server) runPodSandbox(ctx context.Context, req *types.RunPodSandboxRequ
 		return nil, errors.Wrapf(err, resourceErr.Error())
 	}
 
-	if sbox.Config().Linux == nil {
-		sbox.Config().Linux = &types.LinuxPodSandboxConfig{}
-	}
-	if sbox.Config().Linux.SecurityContext == nil {
-		sbox.Config().Linux.SecurityContext = newLinuxSandboxSecurityContext()
```

Comment:
The `newLinuxSandboxSecurityContext` is unused after this commit, and therefore should be removed (by this commit, not a later one).

(Also, I wish there was a way to run linter on every commit -- it would catch issues like this).